planning/behavior_velocity_planner/README.md

```diff
@@ -20,6 +20,7 @@ It consists of several modules. Please refer to the links listed below for detai
 - [Run Out](run-out-design.md)
 - [Speed Bump](speed-bump-design.md)
 - [Out of Lane](out-of-lane-design.md)
+- [Invalid Lanelet](invalid-lanelet-design.md)
```

Comment:
@ahmeddesokyebrahim Thank you for good PR.

I have an opinion on naming. :pray: 
The scene module should preferably be named in a way that is easy for the user to understand its function.
Lanelet is the name for lanelet2 and it is difficult to imagine its function from the user's perspective.
Example:
- no drivable lane/area

Reply:
Thanks @yukkysaito san for your comment and contribution.
I have worked on changing the module name from `invalid_lanelet` to `no_drivable_lane` and reflected that to all [PRs](https://github.com/orgs/autowarefoundation/discussions/3275#discussioncomment-5379038).

As well, you can see a working [video demo](https://youtu.be/XKDQn2bcMPQ) after the name change and you can use [this map file](https://drive.google.com/file/d/1oH9Q0Sk9pm-ttF34Vx-bceQCx9Ja-xmS/view?usp=share_link) for testing if needed.

A reminder again : 
**:exclamation: Merge sequence can be found [here](https://github.com/orgs/autowarefoundation/discussions/3275#discussioncomment-5379038) :exclamation:**

Reply:
@ahmeddesokyebrahim 
Thanks for the correction. :+1: 
LGTM